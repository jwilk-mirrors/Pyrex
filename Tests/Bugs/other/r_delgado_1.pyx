cdef class eggs:

  def __dealloc__(self):
    pass

  def eat(self):
    raise RuntimeError("I don't like that")

