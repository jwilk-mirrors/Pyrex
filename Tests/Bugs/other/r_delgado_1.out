RuntimeError: I don't like that
